Kajabi/sage-lib
# feat(icons): add new icons

## Description

Resolves #915 
Closes #884

Add icons

- [x] -   contact
- [x] -   course
- [x] -   downsell
- [x] -   expand
- [x] -   grant-offer
- [x] -   handle-2
-   ~~mail-1~~
-   ~~mail-2~~

## Screenshots

### After extra mail icon removal
![Screen Shot 2021-10-21 at 3 41 58 PM](https://user-images.githubusercontent.com/24237393/138354052-3a2cfaa9-c574-4866-8740-63d2a8813285.png)


### With extra mail icons

![before-after-1](https://user-images.githubusercontent.com/24237393/138347341-a9865eb6-6d6b-4a38-b035-6029bdcd6389.png)

![before-after-2](https://user-images.githubusercontent.com/24237393/138347351-bcd6957c-c205-46c3-bd42-1adcb76560bf.png)


## Testing in `sage-lib`

1. Navigate to http://localhost:4000/pages/foundations/icon
2. Verify new icons are appearing and old icons are unchanged
3. Navigate to http://localhost:4100/?path=/docs/sage-icon--default
4. Verify new icons are appearing and old icons are unchanged

## Testing in `kajabi-products`
1. (**LOW**) Adding new icons, no old icons updated in this release; nothing to test on `kajabi-products`
